src/sdp/Utils.cpp

```diff
@@ -256,29 +256,29 @@ namespace mediasoupclient
 				{
 					auto& ssrcGroups = *jsonSsrcGroupsIt;
 
-					std::find_if(
```

Comment:
compiler was complaining because `std::find_if` is marked such us the return value must be used. a `for` loop perfectly does the job.